src/services/external/excalidraw-backend/excalidraw.server.ts

```diff
@@ -195,7 +211,11 @@ export class ExcalidrawServer {
         `User '${socket.data.agentInfo.userID}' established connection`,
         LogContext.EXCALIDRAW_SERVER
       );
+
       this.wsServer.to(socket.id).emit(INIT_ROOM);
+
+      this.startCollaboratorModeTimer(socket);
```

Comment:
start the timer after the user has been authorized.
delete the timer also on socket error - there is a handler about it